maxim/logger/gemini/client.py

```
@@ -156,14 +158,14 @@ def send_message_stream(
         return response
 
     def __getattr__(self, name: str) -> Any:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)